cfgov/jinja2/v1/_includes/organisms/email-signup.html

```diff
@@ -28,29 +34,44 @@ <h2 class="{{ 'a-heading' if value.default_heading else 'h3' }} ">
         </h2>
     </header>
 {% endif %}
+{% if value.text %}
+    <p>
+        {{ value.text }}
+    </p>
+{% endif %}
+
+    <form class="o-form o-form__email-signup"
+          id="{{ 'o-form__email-signup_' ~ form_id }}"
+          method="POST"
+          action="/subscriptions/new/"
+          enctype="application/x-www-form-urlencoded">
+        <div class="u-mb15">
+            {% import 'molecules/notification.html' as notification with context %}
+            {{ notification.render('default', false, '') }}
+        </div>
+
+        <div class="m-form-field">
+            <label class="a-label a-label__heading" for="{{ 'email_' ~ form_id }}">
+                Email address
+            </label>
+            <input class="a-text-input a-text-input__full"
+                   id="{{ 'email_' ~ form_id }}"
+                   name="email"
+                   type="email"
+                   placeholder="mail@example.com"
+                   required>
+        </div>
+
+        <div class="m-btn-group">
+            <input class="a-btn" type="submit" value="Sign up">
```

Comment:
Use `<button>` instead of `type="submit"`?

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/4838/commits/75324dfce4ced3b5116c004b5922521ac15ddf4e